src/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar calendar--start-day calendar--month-length">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
remove additional classes on each day, use only modifier classes on calendar

```suggestion
        <div class="calendar__day"></div>
```